### PR TITLE
Add send slack massage to a channel

### PR DESCRIPTION
**Pull request by ekirmayer**
This push adding a functionality to send messages to slacks.
a secret file should exist in the working folder or be specified when calling the function

### CODE REVIEW

**Comment by lioramilbaum** [comment]
@ekirmayer apologies for the long time it took me to review the PR. Can you please provide information on how this service is running?

File: publish/events.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                                          'status': status})
+    events = json.dumps(events.__dict__)
+    events = json.loads(events)
+    for event in events['results']:
+        print(event['name'])
+    meetup.api.Client(getSecret(service='Meetup'))
+    getEvents(status='past')
+
+
+def putPost(service='Slack', channel='sandbox', message='Hello World'):
```
Comment (ShimiTaNaka) [inline publish/events.py]: It's a bit misleading
You allow to pass a service name but statically post to slack?
wouldn't it be wiser to use different "client" class per service
and something of "Factory" pattern?

File: .gitignore [hunk 2]
```diff
@@ -32,6 +34,7 @@ playbooks/*.retry
 
 .vagrant/
 Vagrantfile.local
+.secrets.json
```
Comment (ShimiTaNaka) [inline .gitignore]: if the `.secret*` is in the ignore, why do we need this one as well?

Comment (lioramilbaum) [reply .gitignore]: agree. cleaning it up.

File: publish/events.py (hunk 1 above)
Comment (lioramilbaum) [reply publish/events.py]: Agree it is wiser, but, for MVP isn't it valuable enough? We can afford a technical debt if what right now is valuable enough.

Comment (ShimiTaNaka) [reply publish/events.py]: I agree it doesn't have to be part of MVP, but I do think that if you allow passing service, you should at least throw error is a non-supported service is passed, or, change the method name to indicate that it only handles slack and remove the service input.

Comment (lioramilbaum) [reply publish/events.py]: +1

**Pull request closed without merging**